build.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM golang:1.8
+FROM golang:1.9-alpine
```

Comment:
We think the go 1.9 final will be out way before the trafik 1.4 final is ready, and this feature is the most wanted feature ;) So all in all, we think it's a reasonable risk.